CI/after_success.ps1

```diff
@@ -1,10 +1,15 @@
-if ("$Env:APPVEYOR_REPO_NAME" -ne "Mudlet/Mudlet") {
+if (((Test-Path Env:APPVEYOR) -and "$Env:APPVEYOR_REPO_NAME" -ne "Mudlet/Mudlet") -or
+    ((Test-Path Env:GITHUB_REPOSITORY) -and $Env:GITHUB_REPOSITORY -ne "Mudlet/Mudlet")) {
   exit 0
 }
 
-cd "$Env:APPVEYOR_BUILD_FOLDER\src\release"
+$Script:BuildFolder = If (Test-Path Env:APPVEYOR) { $Env:APPVEYOR_BUILD_FOLDER } Else { "$env:GITHUB_WORKSPACE\b\ninja" };
+
+tree /f $Script:BuildFolder
+
+Set-Location "$Script:BuildFolder\src\release"
 windeployqt.exe --release mudlet.exe
```

Comment:
You may need to review this line after Qt 5.**14** as the `--release` option does not work - it seems to be assumed unless the *opposite* `--debug` option is given. https://bugreports.qt.io/browse/QTBUG-80806 may be relevant for this. Also `--no-virtualkeyboard` might be helpful as well, if we are not including whatever libraries *that* requires...

Reply:
Thanks, good to know